lib/epochs.js

```diff
@@ -119,12 +130,32 @@ function reduceEpochs (ssb, groupId, cb) {
     })
   })
 }
-
 function toMsgURI (id) {
-  if (id.startsWith('%')) return fromMessageSigil(id)
-  else return id
+  return id.startsWith('%') ? fromMessageSigil(id) : id
 }
-function toFeedURI (id) {
-  if (id.startsWith('@')) return fromFeedSigil(id)
-  else return id
+
+function getEpochMembers (ssb, epochId, cb) {
+  const members = new Set()
+  const toExclude = new Set()
+
+  pull(
+    getTangleUpdates.stream(ssb, 'members', epochId),
```

Comment:
aah epochId is the root msg id of the epoch? i guessed that it was the epoch secret. maybe call it epochRoot?

Reply:
epochRoot is a good idea. In practice id === root for tangles, but this id is waaaaay overloaded as a term